### PR TITLE
ATP server UI

### PR DESCRIPTION
**Pull request by Atlante45**

### CODE REVIEW

File: interface/resources/qml/controls-uit/Button.qml [hunk 1]
```diff
@@ -15,8 +15,9 @@ import QtQuick.Controls.Styles 1.4
 import "../styles-uit"
 
 Original.Button {
-    id: button
     property int color: 0
+    property int colorScheme: hifi.colorShemes.light
```
Comment (birarda) [inline interface/resources/qml/controls-uit/Button.qml]: colorShemes?

File: interface/resources/qml/AssetServer.qml [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+import "styles-uit"
+import "controls-uit" as HifiControls
+import "windows-uit"
+import "dialogs"
+
+Window {
+    id: root
+    objectName: "AssetsServer"
+    title: "Assets Server"
```
Comment (birarda) [inline interface/resources/qml/AssetServer.qml]: I'm not sure this title makes sense - shouldn't this be "Asset Server" or just "Assets"?

File: interface/resources/qml/AssetServer.qml [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+                    colorScheme: root.colorScheme
+                    height: 26
+                    width: 26
+
+                    onClicked: root.reload()
+                }
+            }
+
+            Item {
+                // Take the browseButton out of the column flow.
```
Comment (birarda) [inline interface/resources/qml/AssetServer.qml]: deleteButton

File: interface/resources/qml/AssetServer.qml [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+                anchors.left: parent.left
+                anchors.right: parent.right
+                anchors.rightMargin: uploadButton.width + hifi.dimensions.contentSpacing.x
+
+                text: "Add to scene on upload"
+                checked: false
+            }
+
+            Item {
+                // Take the browseButton out of the column flow.
```
Comment (birarda) [inline interface/resources/qml/AssetServer.qml]: uploadButton

File: interface/resources/qml/AssetServer.qml [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+                id: deleteButtonContainer
+                anchors.top: buttonRow.top
+                anchors.right: parent.right
+
+                HifiControls.Button {
+                    id: deleteButton
+                    anchors.right: parent.right
+
+                    text: "DELETE SELECTION"
```
Comment (huffman) [inline interface/resources/qml/AssetServer.qml:110]: why are you yelling?

File: interface/src/Menu.h [hunk 6]
```diff
@@ -35,6 +35,7 @@ namespace MenuOption {
     const QString AnimDebugDrawDefaultPose = "Debug Draw Default Pose";
     const QString AnimDebugDrawPosition= "Debug Draw Position";
     const QString AssetMigration = "ATP Asset Migration";
+    const QString AssetServer = "Asset Server";
```
Comment (huffman) [inline interface/src/Menu.h:38]: Let's change the name of this as well. Not sure what we should call it.. "Asset Browser" ?

**Pull request merged**